packages/wrangler/src/dispatch/index.ts

```diff
@@ -0,0 +1,119 @@
+import { readConfig } from "../config";
```

Comment:
`/dispatch/index.ts` I think should be `/dispatch/dispatch.ts` this is a personal preference.